test/spoc/e2e_spoc_test.go

```diff
@@ -154,13 +150,11 @@ func recordAppArmorTest(t *testing.T) {
 				break
 			}
 		}
-
 		// Run binary...
-		cmd2 := exec.Command(demobinary, "--net-tcp")
+		cmd2 := exec.Command(demobinary, "--net-tcp", "--sleep", "30")
```

Comment:
Why the added sleep here?

Reply:
It was not enough when running the tests. Sometimes needed to run a bit longer.

Reply:
Is this because we now have a race with `ProcessIDByName`? I tried hard to make things work for short-lived processes, so if there is a different reason I'd be happy to dig into things.

Reply:
Yes, that's the reason. It takes a bit of time to find the pid into the list. It is less than 30 seconds, but this is large enough to keep the tests consistently running.